log-manager/src/main/java/io/airlift/log/JsonFormatter.java

```diff
@@ -0,0 +1,45 @@
+package io.airlift.log;
+
+import io.airlift.json.JsonCodec;
+import io.airlift.json.JsonCodecFactory;
+import io.airlift.json.ObjectMapperProvider;
+
+import java.time.Instant;
+import java.util.logging.Formatter;
+import java.util.logging.LogRecord;
+
+public class JsonFormatter
+        extends Formatter
+{
+    private static final JsonCodec<JsonRecord> CODEC = new JsonCodecFactory(new ObjectMapperProvider()).jsonCodec(JsonRecord.class);
+
+    @Override
+    public String format(LogRecord record)
+    {
+        JsonRecord jsonRecord = new JsonRecord(
```

Comment:
The variable could be name just `record` for conciseness.

Reply:
There is already a `record` in that scope

Reply:
Oops. Missed that